constraints/strongStart.js

```diff
@@ -142,4 +142,20 @@ function strongStartClitic(s, ptree, cat){
 	}
 	
 	return vcount;
-}
\ No newline at end of file
+}
+
+function strongStart_catInit(stree, ptree, cat){
```

Comment:
Add a comment indicating the constraint definition in prose, especially which category gets you what result.

Reply:
I think this function's use of cat isn't consistent with the definition that's on the interface info section?

Reply:
Also it's assigning 0 violations on the interface.

Reply:
So I can put "Require Phi Wrapper" back into the interface, but I am a little confused about the GEN settings. If exhaustivity is required, how will we ever get {a (b)}? (where "a" and "b" are prosodic words, {} is an i and () is a phi.) Same issue with "require phi wrapper", the phi wrapper will preclude any situation where an iota has a left daughter which is a word and a right daughter which is a phi.

Reply:
> I think this function's use of cat isn't consistent with the definition that's on the interface info section?

I got confused by the definition: I realize now that the constituency is (every node of category k that is initial)(in a node of category k+2) and not (every node of category k that is initial in a node of category k+2)

Reply:
Right, let me clarify. We actually want to see a violation for {(a (b))}. The constraint isn't intended to check the first daughter of the iota, but the first word that isn't preceded by any other phonological content within the iota, even if it's a "granddaugther" of the iota, so to speak. so {(a (b))} should get a violation because a is the first terminal in the iota and its sister is a phi.

Does that clarify?

I'm wondering at this point if it would be better/easier to implement this constraint as StrongStart-phiMax instead. We can change the phrasing in the handout appropriately.

Reply:
I see...why don't I try to implement that constraint before the meeting and if I don't succeed, you can go with strongStart-phiMax.

Reply:
Sounds good. As I think about this constraint more, I dislike the way it requires a search to arbitrary depths, but it is what it is, and at least it's limited to the beginning of the iota!